### ISSUE TITLE
[iceprod2] load environment from an env-shell.sh (Trac #1652)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1644]
Put in syntax to load from an env-shell.sh in a module.

This is especially important for IPModules, where we need to explicitly insert it before the exe_helper is run.

Since we're touching this code, also add an option for clearing the environment.

Migrated from https://code.icecube.wisc.edu/ticket/1652
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Put in syntax to load from an env-shell.sh in a module.\n\nThis is especially important for IPModules, where we need to explicitly insert it before the exe_helper is run.\n\nSince we're touching this code, also add an option for clearing the environment.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] load environment from an env-shell.sh", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-04-21T21:48:58", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160425T19:11:23: 
* david.schultz commented:

> In changeset 145038:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145038"
> env_shell and env_clear options. fixes #1644
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
